ou_dedetai/installer.py

```diff
@@ -397,9 +396,7 @@ def create_launcher_shortcuts(app: App):
 
     if constants.RUNMODE == 'binary':
         lli_executable = f"{installdir}/{constants.BINARY_NAME}"
-    elif constants.RUNMODE == 'snap':
-        lli_executable = f"{os.getenv('SNAP')}/bin/{constants.BINARY_NAME}"
```

Comment:
This code path wasn't reachable, as this function is only called if RUNMODE is binary. We don't really want to do this otherwise, as we want to be running in a snap